tests/lib/rules/all.js

```diff
@@ -38,34 +30,49 @@ ruleTester.run('lodash.keys', rules.keys, {
   invalid: [{
     code: 'lodash.keys({one: 1, two: 2, three: 3})',
     errors: ['Consider using the native Object.keys()']
-  }, {
-    code: "import keys from 'lodash/keys';",
-    errors: ["Import from 'lodash/keys' detected. Consider using the native Object.keys()"]
-  }, {
-    code: "import keys from 'lodash.keys';",
-    errors: ["Import from 'lodash.keys' detected. Consider using the native Object.keys()"]
-  }, {
-    code: "import { keys } from 'lodash';",
-    errors: ["Import { keys } from 'lodash' detected. Consider using the native Object.keys()"]
   }]
 });
 
+[
+  { fnName: 'keys', alternative: 'Object.keys()' }, 
+  { fnName: 'indexOf', alternative: 'Array.prototype.indexOf()' },
+  { fnName: 'forEach', alternative: 'Array.prototype.forEach()' },
+  { fnName: 'isNaN', ruleName: 'is-nan', alternative: 'Number.isNaN()' }
+].forEach(({ fnName, ruleName, alternative }) => {
```

Comment:
Thanks for the PR. I don't think we need to test 4 rules for `import`. Maybe 1 is enough?

Reply:
@stevemao updated